LoopFollow/Remote/Loop/LoopNightscoutRemoteView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoopNightscoutRemoteView.swift
+//  LoopFollow
+//
+//  Created by Jonas Björkert on 2025-01-14.
+//  Copyright © 2025 Jon Fawcett. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoopNightscoutRemoteView: View {
+    @Environment(\.presentationMode) var presentationMode
+    @ObservedObject var nsAdmin = ObservableUserDefaults.shared.nsWriteAuth
+
+    var body: some View {
+        NavigationView {
+            if !nsAdmin.value {
+                ErrorMessageView(
+                    message: "Please update your token to include the 'admin' role in order to do remote commands."
```

Comment:
add word Loop because this is specific to Loop.
find the equivalent for Trio and add the word Trio (careportal and readable is sufficient)